scripts/package/package-debian.sh

```diff
@@ -97,28 +97,43 @@ create_debian_package(){
 test_debian_package(){
     header "Testing debian package"
     
-    # Set LAST_VERSION_URL to enable upgrade tests
-    export LAST_VERSION_URL="https://dotnetcli.blob.core.windows.net/dotnet/$CHANNEL/Installers/Latest/dotnet-ubuntu-x64.latest.deb"
+    install_bats
+    run_package_integrity_tests
 
+    install_debian_package
+    run_e2e_test
```

Comment:
Do you see an advantage to running E2E twice? Should it just run out of the Native Installers?